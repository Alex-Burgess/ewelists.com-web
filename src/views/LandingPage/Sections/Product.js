import React from "react";
// nodejs library to set properties for components
import PropTypes from "prop-types";
// @material-ui/core components
import withStyles from "@material-ui/core/styles/withStyles";
// @material-ui/icons
import List from "@material-ui/icons/List";
import Perm from "@material-ui/icons/PermIdentity";
import Group from "@material-ui/icons/Group";
// core components
import GridContainer from "components/Grid/GridContainer.js";
import GridItem from "components/Grid/GridItem.js";
import InfoArea from "components/InfoArea/InfoArea.js";

import productStyle from "assets/jss/custom/views/landingPage/productStyle.js";

class LandingProduct extends React.Component {
  render() {
    const { classes } = this.props;
    return (
      <div className={classes.section + " " + classes.sectionGray}>
        <div className={classes.container}>
          <GridContainer justify="center">
            <GridItem xs={12} sm={8} md={8}>
              <h2 className={classes.title}>Get organised with Ewelists</h2>
              <h5 className={classes.description}>
                Create your first Ewelist with the things you know you want, or just the essentials.
              </h5>
            </GridItem>
          </GridContainer>
          <div>
            <GridContainer>
              <GridItem xs={12} sm={4} md={4}>
                <InfoArea
                  className={classes.infoArea}
                  title="Sign Up"
                  description="Sign up using one of your social accounts, or alternatively just an email, to create your first list."
                  icon={Perm}
                  iconColor="primary"
                  vertical
                />
              </GridItem>
              <GridItem xs={12} sm={4} md={4}>
                <InfoArea
                  className={classes.infoArea}
                  title="Add Items"
                  description="Add items by copying the link to the product, or alternatively use one of our ready made lists if you're looking for inspiration."
                  icon={List}
                  iconColor="rose"
                  vertical
                />
              </GridItem>
              <GridItem xs={12} sm={4} md={4}>
                <InfoArea
                  className={classes.infoArea}
                  title="Share"
                  description="Share your first ewelist with friends and family."
                  icon={Group}
                  iconColor="info"
                  vertical
                />
              </GridItem>
            </GridContainer>
          </div>
        </div>
      </div>
    );
  }
}

LandingProduct.propTypes = {
  classes: PropTypes.object
};

export default withStyles(productStyle)(LandingProduct);
