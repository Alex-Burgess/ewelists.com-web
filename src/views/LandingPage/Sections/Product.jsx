/*!

=========================================================
* Material Kit PRO React - v1.7.0
=========================================================

* Product Page: https://www.creative-tim.com/product/material-kit-pro-react
* Copyright 2019 Creative Tim (https://www.creative-tim.com)

* Coded by Creative Tim

=========================================================

* The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

*/
import React from "react";
// nodejs library to set properties for components
import PropTypes from "prop-types";
// @material-ui/core components
import withStyles from "@material-ui/core/styles/withStyles";

// @material-ui/icons
import List from "@material-ui/icons/List";
import Perm from "@material-ui/icons/PermIdentity";
import Group from "@material-ui/icons/Group";
// core components
import GridContainer from "components/Grid/GridContainer.jsx";
import GridItem from "components/Grid/GridItem.jsx";
import InfoArea from "components/InfoArea/InfoArea.jsx";

import productStyle from "assets/jss/custom/views/landingPage/productStyle.jsx";

class LandingProduct extends React.Component {
  render() {
    const { classes } = this.props;
    return (
      <div className={classes.section + " " + classes.sectionGray}>
        <div className={classes.container}>
          <GridContainer justify="center">
            <GridItem xs={12} sm={8} md={8}>
              <h2 className={classes.title}>Get organised with Ewelists</h2>
              <h5 className={classes.description}>
                Create your first Ewelist with the things you know you want, or just the essentials.
              </h5>
            </GridItem>
          </GridContainer>
          <div>
            <GridContainer>
              <GridItem xs={12} sm={4} md={4}>
                <InfoArea
                  className={classes.infoArea}
                  title="Sign Up"
                  description="Sign up using one of your social accounts, or alternatively just an email, to create your first list."
                  icon={Perm}
                  iconColor="primary"
                  vertical
                />
              </GridItem>
              <GridItem xs={12} sm={4} md={4}>
                <InfoArea
                  className={classes.infoArea}
                  title="Add Items"
                  description="Add items by copying the link to the product, or alternatively use one of our ready made lists if you're looking for inspiration."
                  icon={List}
                  iconColor="rose"
                  vertical
                />
              </GridItem>
              <GridItem xs={12} sm={4} md={4}>
                <InfoArea
                  className={classes.infoArea}
                  title="Share"
                  description="Share your first ewelist with friends and family."
                  icon={Group}
                  iconColor="info"
                  vertical
                />
              </GridItem>
            </GridContainer>
          </div>
        </div>
      </div>
    );
  }
}

LandingProduct.propTypes = {
  classes: PropTypes.object
};

export default withStyles(productStyle)(LandingProduct);
