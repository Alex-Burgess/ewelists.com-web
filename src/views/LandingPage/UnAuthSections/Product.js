import React from "react";
// @material-ui/core components
import { makeStyles } from "@material-ui/core/styles";
// @material-ui/icons
import List from "@material-ui/icons/List";
import Perm from "@material-ui/icons/PermIdentity";
import Group from "@material-ui/icons/Group";
// core components
import GridContainer from "components/Grid/GridContainer.js";
import GridItem from "components/Grid/GridItem.js";
import InfoArea from "components/InfoArea/InfoArea.js";

import styles from "assets/jss/custom/views/landingPage/productStyle.js";
const useStyles = makeStyles(styles);

export default function LandingProduct(props) {
  const classes = useStyles();

  return (
    <div className={classes.section + " " + classes.sectionGray}>
      <div className={classes.container}>
        <GridContainer justify="center">
          <GridItem xs={12} sm={8} md={8}>
            <h2 className={classes.title}>Get organised with Ewelists</h2>
            <h5 className={classes.description}>
              Create your first Ewelist with the things you know you want, or just the essentials.
            </h5>
          </GridItem>
        </GridContainer>
        <div>
          <GridContainer>
            <GridItem xs={12} sm={4} md={4}>
              <InfoArea
                className={classes.infoArea}
                title="Sign Up"
                description="Sign up using one of your social accounts, or alternatively just an email, to create your first list."
                icon={Perm}
                iconColor="primary"
                vertical
              />
            </GridItem>
            <GridItem xs={12} sm={4} md={4}>
              <InfoArea
                className={classes.infoArea}
                title="Add Items"
                description="Add items by copying the link to the product, or alternatively use one of our ready made lists if you're looking for inspiration."
                icon={List}
                iconColor="rose"
                vertical
              />
            </GridItem>
            <GridItem xs={12} sm={4} md={4}>
              <InfoArea
                className={classes.infoArea}
                title="Share"
                description="Share your first ewelist with friends and family."
                icon={Group}
                iconColor="info"
                vertical
              />
            </GridItem>
          </GridContainer>
        </div>
      </div>
    </div>
  );
}
