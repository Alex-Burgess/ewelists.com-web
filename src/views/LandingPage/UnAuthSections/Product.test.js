import * as React from 'react';
import TestRenderer from 'react-test-renderer';
import Product from './Product';

describe('Tests for Product section.', () => {
  test('Renders product section', () => {
    const tree = TestRenderer
      .create(<Product />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
