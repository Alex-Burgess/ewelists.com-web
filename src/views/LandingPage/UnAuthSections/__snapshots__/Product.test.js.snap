// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for Product section. Renders product section 1`] = `
<div
  className="makeStyles-section-2 makeStyles-sectionGray-3"
>
  <div
    className="makeStyles-container-1"
  >
    <div
      className="MuiGrid-root makeStyles-grid-7  MuiGrid-container MuiGrid-justify-xs-center"
    >
      <div
        className="MuiGrid-root makeStyles-grid-111  MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-8 MuiGrid-grid-md-8"
      >
        <h2
          className="makeStyles-title-4"
        >
          Get organised with Ewelists
        </h2>
        <h5
          className="makeStyles-description-5"
        >
          Create your first Ewelist with the things you know you want, or just the essentials.
        </h5>
      </div>
    </div>
    <div>
      <div
        className="MuiGrid-root makeStyles-grid-7  MuiGrid-container"
      >
        <div
          className="MuiGrid-root makeStyles-grid-111  MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 MuiGrid-grid-md-4"
        >
          <div
            className="makeStyles-infoArea-112 makeStyles-infoArea-6"
          >
            <div
              className="makeStyles-iconWrapper-113 makeStyles-primary-114 makeStyles-iconWrapperVertical-125"
            >
              <svg
                aria-hidden="true"
                className="MuiSvgIcon-root makeStyles-icon-121 makeStyles-iconVertical-126"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 5.9c1.16 0 2.1.94 2.1 2.1s-.94 2.1-2.1 2.1S9.9 9.16 9.9 8s.94-2.1 2.1-2.1m0 9c2.97 0 6.1 1.46 6.1 2.1v1.1H5.9V17c0-.64 3.13-2.1 6.1-2.1M12 4C9.79 4 8 5.79 8 8s1.79 4 4 4 4-1.79 4-4-1.79-4-4-4zm0 9c-2.67 0-8 1.34-8 4v3h16v-3c0-2.66-5.33-4-8-4z"
                />
              </svg>
            </div>
            <div
              className="makeStyles-descriptionWrapper-122"
            >
              <h4
                className="makeStyles-title-123"
              >
                Sign Up
              </h4>
              <div
                className="makeStyles-description-124"
              >
                Sign up using one of your social accounts, or alternatively just an email, to create your first list.
              </div>
            </div>
          </div>
        </div>
        <div
          className="MuiGrid-root makeStyles-grid-111  MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 MuiGrid-grid-md-4"
        >
          <div
            className="makeStyles-infoArea-112 makeStyles-infoArea-6"
          >
            <div
              className="makeStyles-iconWrapper-113 makeStyles-rose-119 makeStyles-iconWrapperVertical-125"
            >
              <svg
                aria-hidden="true"
                className="MuiSvgIcon-root makeStyles-icon-121 makeStyles-iconVertical-126"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M3 13h2v-2H3v2zm0 4h2v-2H3v2zm0-8h2V7H3v2zm4 4h14v-2H7v2zm0 4h14v-2H7v2zM7 7v2h14V7H7z"
                />
              </svg>
            </div>
            <div
              className="makeStyles-descriptionWrapper-122"
            >
              <h4
                className="makeStyles-title-123"
              >
                Add Items
              </h4>
              <div
                className="makeStyles-description-124"
              >
                Add items by copying the link to the product, or alternatively use one of our ready made lists if you're looking for inspiration.
              </div>
            </div>
          </div>
        </div>
        <div
          className="MuiGrid-root makeStyles-grid-111  MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 MuiGrid-grid-md-4"
        >
          <div
            className="makeStyles-infoArea-112 makeStyles-infoArea-6"
          >
            <div
              className="makeStyles-iconWrapper-113 makeStyles-info-118 makeStyles-iconWrapperVertical-125"
            >
              <svg
                aria-hidden="true"
                className="MuiSvgIcon-root makeStyles-icon-121 makeStyles-iconVertical-126"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M16 11c1.66 0 2.99-1.34 2.99-3S17.66 5 16 5c-1.66 0-3 1.34-3 3s1.34 3 3 3zm-8 0c1.66 0 2.99-1.34 2.99-3S9.66 5 8 5C6.34 5 5 6.34 5 8s1.34 3 3 3zm0 2c-2.33 0-7 1.17-7 3.5V19h14v-2.5c0-2.33-4.67-3.5-7-3.5zm8 0c-.29 0-.62.02-.97.05 1.16.84 1.97 1.97 1.97 3.45V19h6v-2.5c0-2.33-4.67-3.5-7-3.5z"
                />
              </svg>
            </div>
            <div
              className="makeStyles-descriptionWrapper-122"
            >
              <h4
                className="makeStyles-title-123"
              >
                Share
              </h4>
              <div
                className="makeStyles-description-124"
              >
                Share your first ewelist with friends and family.
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
